stripper/ze_star_wars_prequel_v1.cfg

```
@@ -36,4 +36,18 @@ modify:
 		"OnMapSpawn" "zm_respawn_killAddOutputmins -48 -48 -641-1"
 		"OnMapSpawn" "zm_respawn_killAddOutputmaxs 48 48 641-1"
 	}
-}
\ No newline at end of file
+}
+
+;Fix tp avoidence spot on level 1
+add:
+{
+	"UseLandmarkAngles" "1"
+	"targetname" "tp_skip_1"
+	"target" "desti_zm02"
+	"StartDisabled" "0"
+	"spawnflags" "4097"
+	"origin" "-11767.5 -10208.5 -11255"
+	"model" "*273"
+	"CheckDestIfClearForPlayer" "0"
+	"classname" "trigger_teleport"
+}
```

```suggestion
;Fix tp avoidence spot on level 1
add:
{
	"UseLandmarkAngles" "1"
	"targetname" "tp_zm01"
	"target" "desti_zm01"
	"StartDisabled" "1"
	"spawnflags" "4097"
	"origin" "-12616 -12776 -10848"
	"model" "*272"
	"CheckDestIfClearForPlayer" "0"
	"classname" "trigger_teleport"
}
```
I'm not sure why you've made this its own teleport rather than integrate it with the existing tp_zm01 logic. As it stands, your implementation covers a *significant* portion of the spawn area and will end up teleporting basically everyone away immediately since the teleport starts enabled.